spec/components/breadcrumbs_spec.rb

```diff
@@ -20,6 +20,18 @@ def assert_link_with_text_in(selector, link, text)
     assert_link_with_text_in('.gem-c-breadcrumbs--item:first-child', '/section', 'Section')
   end
 
+  it "renders schema data" do
+    breadcrumbs = [{ title: 'Section', url: '/section' }]
```

Comment:
Can you add a test with 2 elements and `#is_current_page` as well?

Reply:
Added!